acvm-repo/blackbox_solver/Cargo.toml

```diff
@@ -33,6 +34,8 @@ p256 = { version = "0.11.0", features = [
     "digest",
     "arithmetic",
 ] }
+grumpkin = { git = "https://github.com/noir-lang/grumpkin", features = ["std"] }
```

Comment:
We should open up an issue to use the arkworks version; back when this was made they did not have grumpkin, they recently added it a month ago

Reply:
Not sure if all of the parameters match, so might be better to make things work with our version which we can modify if it doesn't match

Reply:
Yeah, we can't do this at the minute. arkworks haven't pushed this crate to crates.io and using a git dependency results in issues due to it wanting a different version of `ark-bn254`